go/client/cmd_chat_hide.go

```diff
@@ -0,0 +1,118 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/chat1"
+)
+
+type CmdChatHide struct {
+	libkb.Contextified
+	resolver chatCLIConversationResolver
+	status   chat1.ConversationStatus
+}
+
+func newCmdChatHide(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "hide",
+		Usage:        "Hide or block a conversation.",
+		ArgumentHelp: "[<conversation>]",
+		Action: func(c *cli.Context) {
+			cmd := &CmdChatHide{Contextified: libkb.NewContextified(g)}
+			cl.ChooseCommand(cmd, "hide", c)
+		},
+		Flags: []cli.Flag{
```

Comment:
Would you mind adding "block" and "unhide" into `chat_cli_flag_utils.go`, and do the following here instead? I think it's easier to maintain and reuse code to have all them in the same place. But it could be just me.

```
Flags: append(getConversationResolverFlags(), mustGetChatFlags("block", "unhide")...),
```